content/exercises/data-scenarios/raster-needs-reformatting/index.md

```diff
@@ -18,7 +18,8 @@ A colleague e-mails you aerial imagery that they extracted from the IceBridge DM
 dataset [@dominguez_2010_icebridge_dms] of a potential study site that you would like to
 view in QGIS alongside some other data.
 
-![datafile.jpg](/content/exercises/data-scenarios/raster-needs-reformatting/datafile.jpg){width="65%" fig-align="center"}
+![datafile.jpg](/content/exercises/data-scenarios/raster-needs-reformatting/datafile.jpg
+){width="65%" fig-align="center"}
```

Comment:
After renaming this file to `.md`, the format checker started complaining about the length of this line. This is ugly, but there seems to be no way to ignore format errors with the tool we selected.